release_creation/snapshot/requirements/v1.4.pre.requirements.txt

```diff
@@ -6,7 +6,7 @@ dbt-postgres~=1.4.0b1
 dbt-spark[PyHive,ODBC]~=1.4.0b1
 dbt-databricks~=1.4.0
 dbt-trino~=1.4.0
-dbt-rpc~=0.1.1 --pre
+dbt-rpc~=0.1.3b1
```

Comment:
Just to make sure I understand how `~=` works here, we're effectively moving the lower bound up from 0.1.1 to 0.1.3b and changing nothing else?